dev_docs/contributing/how_we_use_github.mdx

```diff
@@ -205,10 +205,12 @@ it's notable and affects our users) and which section it appears in. For example
 - `release_node:plugin_api_changes`: Specifies a changes to the plugin API and adds the PR to the Plugin API changes page in the Developer Guide
 - `release_note:skip`: Omits the PR from release notes
 
-These labels related to backporting PRs:
+The following labels are related to backporting PRs:
 
 - `auto-backport`: Automatically backport this PR (to the branches related to
-   version labels) after it's merged
-- `backport`: This PR was backported
-- `backport:skip`: This PR does not require backporting
-
+   version labels) after it's merged. Requires adding desired target versions labels.
+- `backport:prev-minor`: Automatically backport to one branch/version back from main.
+- `backport:prev-major`: Automatically backport to whichever branch contains the next patch version of the previous major release.
+- `backport:all-open`: Automatically backport to all open branches/versions. This is functionally equivalent to backport:prev-major at the time of writing.
```

Comment:
It is unclear to me what is meant by "open". We can (and sometimes do) backport docs changes very far back across many versions, but I assume this isn't the intention?:
```suggestion
- `backport:all-open`: Automatically backport to all open branches/versions. This is functionally equivalent to backport:prev-major at the time of writing.
```